Exiled.Events/Patches/Generic/ProcessQuery.cs

```diff
@@ -33,6 +33,11 @@ private static bool CheckPermissions(ICommand command, CommandSender sender)
                 sender.RaReply($"{command.Command}#You do not have permissions to use this command. Required permission: {permissionCommand.Permission}", false, true, string.Empty);
                 return false;
             }
+            else if (command is IGamePermissionCommand gamePermissionCommand && !sender.CheckPermission(gamePermissionCommand.Permission))
+            {
+                sender.RaReply($"{command.Command}#You do not have permissions to use this command. Required permission: PlayerPermissions.{gamePermissionCommand.Permission}", false, true, string.Empty);
```

Comment:
It won't display required permissions because it's an array, use something like `string.join(", ", permissions)